usagov/usagov-2021
# USAGOV-1992: No Raw Unicode

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1992

## Description

Don't encode accented characters as unicode entities in JSON datalayer output for taxonomy terms.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

- Go through the of steps in a Spanish scam wizard. At each step view source to inspect the datalayer payload. Should not have raw unicode sequences, you should see accented letters in the Taxonomy_Text_* values:

```
    "Taxonomy_Text_1": "Home",
    "Taxonomy_Text_2": "Estafas y fraudes",
    "Taxonomy_Text_3": "Aprenda dónde reportar o denunciar una estafa",
    "Taxonomy_Text_4": "¿Dónde ocurrió la estafa?",
    "Taxonomy_Text_5": "¿Qué categoría describe mejor el tipo de estafa que experimentó?",
    "Taxonomy_Text_6": "¿Qué categoría describe mejor el tipo de estafa que experimentó?",
```

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
